src/main/java/me/xemor/configurationdata/entity/EntityData.java

```diff
@@ -73,7 +75,13 @@ protected EntityData(ConfigurationSection configurationSection) {
 
     private EntityData(EntityType entityType) {
         this.entityType = entityType;
+
+        nameTag = null;
         shouldDespawn = true;
+        customNameVisible = true;
+        silent = false;
+        visualFire = true;
```

Comment:
I don't think the default for visual fire should be true, doesn't that mean the entity is on fire visually? lmao

Reply:
Oh you're totally right

Reply:
thank god I was reading about that the other day. crisis averted